samples/Mvc.Client/Properties/launchSettings.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Consider removing this unrelated change. We'll maybe include it in the future, but it will be part of another PR :clap: